kyo-sttp/shared/src/main/scala/kyo/internal/KyoSttpMonad.scala

```diff
@@ -21,7 +21,15 @@ object KyoSttpMonad:
             protected def handleWrappedError[T](rt: T < Fibers)(
                 h: PartialFunction[Throwable, T < Fibers]
             ) =
-                IOs.catching(rt)(h)
+                IOs.catching(rt) {
+                    case ex if ex eq Fibers.Interrupted =>
+                        Fibers.interrupted
+                    case ex if h.isDefinedAt(ex) =>
+                        h(ex)
+                }
+
+            override def handleError[T](rt: => T < Fibers)(h: PartialFunction[Throwable, T < Fibers]) =
```

Comment:
This method is used internally by sttp. I had to override it as well to avoid the translation in some codepaths.